requirements.txt

```
@@ -66,8 +66,8 @@ jinja2==3.0.1; python_version >= '3.6'
 jmespath==0.10.0; python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2, 3.3'
 jsonschema==3.2.0
 kombu==4.6.11; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
+libcloudforensics==20211014
```

to include latest changes
```suggestion
libcloudforensics==20211027
```